packages/remix-dev/compiler/utils/tsconfig/write-config-defaults.ts

```diff
@@ -65,7 +68,9 @@ export function writeConfigDefaults(configPath: string) {
   let requiredChanges = [];
 
   if (!("include" in fullConfig)) {
-    config.include = ["remix.env.d.ts", "**/*.ts", "**/*.tsx"];
+    config.include = configType === "jsconfig.json"
+      ? ["**/*.js", "**/*.jsx"]
+      : ["remix.env.d.ts", "**/*.ts", "**/*.tsx"];
     suggestedChanges.push(
```

Comment:
The shown message should also be changed here

Reply:
oops, iirc I did update it in a diff commit before I reverted some things as I merged https://github.com/remix-run/remix/pull/3936 to keep these changes small